fourth-wall.css

```diff
@@ -81,6 +81,9 @@ body {
 #pulls li.thumbsup {
     background: #5c9317;
 }
+#pulls li.changes-requested {
+    background: #BF1E2D !important;
```

Comment:
The `!important` thing was a bit weird. The class that get's applied if the PR has been "thumbsed up" has the `!important` attribute applied to it if you inspect it in the page. I couldn't find where it was coming from in the fourth wall repo or it's css. Any ideas?